multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -359,12 +312,16 @@ write_files:
                       "-initial-cluster-token",
                       "skydns-etcd"
                     ],
-                    "image": "gcr.io/google_containers/etcd:2.0.9",
+                    "image": "gcr.io/google_containers/etcd:2.2.1",
```

Comment:
I tested this without the architecture and it worked - I wanted to keep the diff as small as possible. However, I'm pretty sure the arch won't hurt.